### DISCUSSION TITLE
3d mesh to real object registration

### DISCUSSION THREAD

**Discussion by jacopomstrada** [isl-org/Open3D#5832] [Q&A]
Given a 3D model mesh, I would like to perform registration between the mesh itself and a 3D print of the same model, which I obtain the point cloud from through a Kinect sensor. I have tried to obtain the point cloud from the mesh and then perform global registration, but the result is frequently incorrect (not always considering the randomicity of RANSAC). I would like to have some suggestions on how to approach the problem.

Thanks in advance

**Comment by hllllkkkkkk** [comment]
So depending on the object, you may limit the transformation matrix itself. Such as, if your object is more likely to be a spherical object, than probably ransac & icp will fails due to lack of features on sides & faces. So, after doing initial transformation, you may limit rotation angles & positions to make ransac choose random points in a more suitable way.
